modules/bbp-core/src/main/resources/contexts/bbp/core/mediatype/v0.1.0.json

```diff
@@ -0,0 +1,54 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/mediatype/v0.1.0",
+    {
+      "ianaform": "https://www.iana.org/form/media-types/",
+      "typeName": {
+        "@id": "ianaform:typeName"
+      },
+      "subtype": {
+        "@id": "ianaform:subtype"
+      },
+      "requiredParameter": {
+        "@id": "ianaform:requiredParameter"
+      },
+      "optionalParameter": {
+        "@id": "ianaform:optionalParameter"
+      },
+      "encoding": {
+        "@id": "ianaform:encoding"
+      },
+      "security": {
+        "@id": "ianaform:security"
+      },
+      "Interoperability": {
```

Comment:
Should **interoperability** be lower case?

Reply:
Yes.